certified-connectors/Waaila/apiDefinition.swagger.json

```diff
@@ -0,0 +1,881 @@
+                        "datasourceId": {
+                          "type": "string",
+                          "description": "datasourceId"
+                        },
+                        "name": {
+                          "type": "string",
+                          "description": "name"
+                        },
+                        "description": {
+                          "type": "string",
+                          "description": "description"
+                        },
+                        "tests": {
+                          "type": "string",
+                          "description": "tests"
+                        },
+                        "providerCode": {
+                          "type": "string",
+                          "description": "providerCode"
+                        },
+                        "testsCount": {
+                          "type": "integer",
+                          "format": "int32",
+                          "description": "testsCount"
+                        },
+                        "version": {
+                          "type": "string",
+                          "description": "version"
+                        },
+                        "taglist": {
+                          "type": "array",
+                          "items": {},
+                          "description": "taglist"
+                        },
+                        "etag": {
+                          "type": "string",
+                          "description": "etag"
+                        },
+                        "modifiedAt": {
+                          "type": "string",
+                          "description": "modifiedAt"
+                        },
+                        "createdAt": {
+                          "type": "string",
+                          "description": "createdAt"
+                        },
+                        "guid": {
+                          "type": "string",
+                          "description": "guid"
+                        }
+                      }
+                    },
+                    "description": "testsuites"
+                  },
+                  "etag": {
+                    "type": "string",
+                    "description": "etag"
+                  },
+                  "modifiedAt": {
+                    "type": "string",
+                    "description": "modifiedAt"
+                  },
+                  "createdAt": {
+                    "type": "string",
+                    "description": "createdAt"
+                  },
+                  "guid": {
+                    "type": "string",
+                    "description": "guid"
+                  }
+                }
+              }
+            }
+          }
+        },
+        "x-ms-visibility": "important"
+      }
+    },
+    "/v1/depot/3db88c3b-fd01-414c-a2aa-90387a54b350/testsuite/5b3047c7-b05b-4049-91a2-6133b333c204": {},
```

Comment:
@matesmac Please remove this empty operation from the swagger as it won't add any functionality to the connector.

Reply:
@sriyen-msft Removed